nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -193,6 +193,14 @@ def setup_experiment(self, config):
                               params_sparse, nonzero_params_sparse2,
                               float(nonzero_params_sparse2) / params_sparse)
 
+        modify_init_hooks = config.get("modify_init_hooks", [])
```

Comment:
Keep in mind that by creating a list of init hooks that are called in sequence, you may have one init hook overriding the other

Reply:
True. Personally, I find it find that this is the case. Part of my goal was more flexibility for the caller as I was encountering a few scenarios that offered post-init modifications - namely, loading from multiple checkpoints and, separately, evolving the init via MetaInit.